aten/src/ATen/native/vulkan/glsl/add.glsl

```diff
@@ -9,7 +9,9 @@ layout(set = 0, binding = 0) uniform PRECISION restrict writeonly image3D   uOut
 layout(set = 0, binding = 1) uniform PRECISION                    sampler3D uInput0;
 layout(set = 0, binding = 2) uniform PRECISION                    sampler3D uInput1;
 layout(set = 0, binding = 3) uniform PRECISION restrict           Block {
-  ivec3 size;
+  ivec4 size;
+  ivec4 isize0;
+  ivec4 isize1;
   float alpha;
```

Comment:
Can this be packed more tightly?  I'm thinking:

ivec3 size
float alpha
ivec4 isize0
ivec4 isize1

If this works, same for others in this PR as well.

Reply:
I'll check again, but I think there were some issues with the alignment (as we've seen before when using vec3)